### DISCUSSION TITLE
Suspended Account

### DISCUSSION THREAD

**Discussion by Vaishnavipatel26** [community/community#135697] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Account Name: UDAIBHAT
link to profile : https://github.com/UDAIBHAT

My Account has been suspended and i am unable to contact support through that acoount due access denial , so i am contacting you through this secodary account , Please help me out as i am unable to access my account

i have also recieved the password reset link from github stating that my account has undergone a suspecious login activity resulting in freezing my account , but through the link provided i am unable to reset my password as it is showing "Access to your account has been suspended due to voilation of terms and condition "

I have also created the two Tickets in github support but its been a week and got no reply from github
Ticket 1 : #2896220
Ticket 2 : #2898013

Plz help me out my work is on github as i am a student with placements on ogoing i am unable to show my projects Please help me out

Neither passwork reset is working nor recieving any reply from support PLEASE HELP please

Its been 4 weeks since i have posted the tickets there is no reply from github . My Whole is suspended because of you , i am facing a lot loss in carrerr rightnow Please help me out 

plz review my ticket atleast make contact with these emails

udaibhat181@gmail.com
patelvaishnavim@gmail.com

please review my  discussion as alot of people are struglling from it

**Comment by still-hash-ir** [comment]
I am facing the same issue and in the middle of a job search. It sucks! Hope they do something about this.

**Reply by Vaishnavipatel26** [reply]
mine tooo i have also placements going on and my projects are on github even i dont have on local machine

**Reply by still-hash-ir** [reply]
Hey, my account has been restored. Support said that their team is currently experiencing a higher volume of inquiries than usual, which has resulted in longer response times. They have been investigating suspicious activity and account hijacking. As a security measure, they have placed restrictions on some accounts to combat this. Don't lose hope, it took them some time but they are reviewing every support ticket. Hopefully, you will get access too!

**Reply by Vaishnavipatel26** [reply]
ok i am for them to respond on this issue

**Reply by HaroldTopre** [reply]
me too I'm still waiting for their response

**Reply by DTH-5** [reply]
@haroldstest how many days it took to reply your ticket?

**Reply by HaroldTopre** [reply]
> @haroldstest how many days it took to reply your ticket?

Unfortunately They haven't replied yet... I submitted my ticket 2 months ago

**Reply by komascript** [reply]
> > @haroldstest how many days it took to reply your ticket?
> 
> Unfortunately They haven't replied yet... I submitted my ticket 2 months ago

Meanwhile @haroldstest is visible again.

**Reply by Haroldfromk** [reply]
> > > @haroldstest how many days it took to reply your ticket?
> > 
> > 
> > Unfortunately They haven't replied yet... I submitted my ticket 2 months ago
> 
> Meanwhile @haroldstest is visible again.

It took over 2months

**Reply by tsukiyoz** [reply]
my account is suspended too, what i can do is use friend's github account seek help!
my account username is codegoz，github.com/codegoz , i don't know why i was suspended and when i come to github community to submit a work order，it needs SMS code，i'm a chinese coder，there is no China +86 code.

above all，how can i restore my account, can any one help me?

you can keep contact with me by 
tsukiyo6@163.com

**Comment by ygetelcare** [comment]
Same here, apparently just connecting CoPilot in VSCode will trigger some bug in their violation screening and get accounts suspended, it happened to a new account I created, yge2020, just for trying CoPilot.  How ridiculous, this is technically fraudulent business practice, we need to report them to the BBB.

**Comment by Haroldfromk** [comment]
Today I got message from Github Support

and Finally I get my account but it took over 2 months....

Hope you guys get your accounts back ASAP

**Comment by komascript** [comment]
The problem seems to be occurring more frequently at the moment. Yesterday my account received a “Password reset notification due to suspicious activity” in the middle of a release process. And this despite the fact that my account has been two-factor protected for a long time. I managed to create a new password and have been able to log in again since then. However, my account remains flagged and all my repositories are only visible to me (although the visibility is set to “public”). A ticket has been created. But if the reactivation takes a long time, this will unfortunately have an extremely negative impact on development. At the moment, users cannot even report problems. I'm already thinking about moving again.

**Comment by centxv** [comment]
Dear GitHub Support Team,

I hope this message finds you well. I am writing to follow up on my account suspension case, as it has been taking longer than expected, and I am increasingly concerned about the impact on my work.

I initially applied for account recovery because my account was suspended, and I am uncertain of the exact reason. After receiving a notification that my account was accessed by an unauthorized person, which I was unaware of, the suspension occurred. This situation has significantly hindered my ability to continue my business, projects, and especially my freelance work, which has been severely affected by this incident.

I kindly request that you review my case as soon as possible. Resolving this matter promptly would greatly help me get back to my work and projects without further delays.

Thank you for your attention to this urgent matter.
suspended account: https://github.com/cent-dxv

Best regards,

**Reply by tsukiyoz** [reply]
Dear github support team
My account was suspended too, and as a chinese coder there is no way to contact to github team. no Chinese SMS code can be send, can't contact by email because of region restrictions.
I kindly hope you review my case as soon as possible.
[github.com/codegoz](github.com/codegoz)

**Comment by komascript** [comment]
My account is back. What I've done:

- Password recovery process to setup a new password → I got access to @komascript again, but it stayed invisible for all others.
- Reactivation of the 2FAs one by one (retract one, add it again, retract next, add it again …)
- Check the security log → there wasn't any suspicious entry
- Check if there is a new repository in @komascript → no
- Check if there are any suspicious commits → no
- Open a ticket for account recovery, telling that I've done the steps above and asking how to get back the visibility of my account.
- Waiting
- After 10 days I got an answer to the ticket: I should check the security log and remove suspicious repositories.
- For security done the check steps again → found nothing suspicious (as expected).
- Answered, that I've done the steps again and everything is fine. Asked for full reactivation of my account again.
- Today the flag is removed and @komascript is visible for all again.

Hope you will get back the full function of your accounts and repositories soon, too.

**Comment by centxv** [comment]
first of all, I want to say congra i know how stressful it is 
how did you get your account back i didn't get it. 
mine is almost 2 months since it's suspended 
@komascript

**Reply by komascript** [reply]
> how did you get your account back

I've listed all the steps in my previous reply. Start with the process for lost passwords to setup a new one. Then do all the other steps from your original account. If you currently have only one second factor activate a new one in the second step, before you retract the existing one.

**Reply by centxv** [reply]
> > how did you get your account back
> 
> I've listed all the steps in my previous reply. Start with the process for lost passwords to setup a new one. Then do all the other steps from your original account. If you currently have only one second factor activate a new one in the second step, before you retract the existing one.

yeah I tried to recover my password but right after i passed the captcha and hit the send email button it didn't send 
but it dose with this account that why i asked

**Reply by centxv** [reply]
![image](https://github.com/user-attachments/assets/8d6e02ce-2fe6-45f5-a3a2-dcd0d575a9c6)

**Reply by komascript** [reply]
Click to the green button "Send password reset email".

**Reply by centxv** [reply]
yeah I did but it doesn't send the recovery password link. |

**Reply by centxv** [reply]
it dosn't even acknowledge that the email is sent ti dose simply redirects to the home page. 
i tried to register a new one with my email associated with the lost account to chek if my data still exists. 
yes it dose it dose say this email is associated with another account

**Reply by komascript** [reply]
Have you checked your SPAM folder?

I can only tell what helped in my case. If this does not help in your case, I cannot help. Sorry.

But surely, if you cannot get a password and therefore cannot open a ticket from within this account, it will be hard to get back the account. How should the support know that you are really the owner?

**Reply by Haroldfromk** [reply]
Have you ever sent tickets using your origin account ?
(I mean suspended account, you can send ticket using your origin account regardless of suspendid account), 
and your sub account (your currnent account, when you send ticket plz talk about your ticket number (written by your origin account)

In my case, the support manager checked my tickets (written by my another account, not origin account) and then he read my origin ticket.

**Reply by komascript** [reply]
> Have you ever sent tickets using your origin account ?

Yes.

**Reply by Haroldfromk** [reply]
> > Have you ever sent tickets using your origin account ?
> 
> Yes.

How about this account (komascript) ?
Did this account send ticket too??

**Reply by komascript** [reply]
I opened the ticket *only* from this account, because this was the flagged account. IMHO it helps the support if you open a ticket from the affected account. Nevertheless in the ticket you should always also name the affected account. And IMHO you should not flood the support with several tickets.

**Reply by Haroldfromk** [reply]
> I opened the ticket _only_ from this account, because this was the flagged account. IMHO it helps the support if you open a ticket from the affected account. Nevertheless in the ticket you should always also name the affected account. And IMHO you should not flood the support with several tickets.

In my case, I sent another ticket to see my origin ticket, and they replied my another ticket.
I totally agree with you, but I had to because at that time I had ongoing project

**Comment by JoshuaDevCo** [comment]
Same for me, it's over 3 months now that I can't access anything. I'm really losing hope!

**Comment by Michel4119** [comment]
I am facing the same issue and in the end of a job search. It sucks[!](https://xnxubdvpn.us/) Hope they do something about this.

**Reply by ygetelcare** [reply]
They finally got back to my ticket after 3 months telling me it was merely because my credit card got declined when trying out the Copilot, how ridiculous is that????  The risk of losing everything is just too high with their reckless rules and paranoia.

**Reply by JoshuaDevCo** [reply]
> They finally got back to my ticket after 3 months telling me it was merely because my credit card got declined when trying out the Copilot, how ridiculous is that???? The risk of losing everything is just too high with their reckless rules and paranoia.

mongoloids they are

**Comment by JoshuaDevCo** [comment]
they dont care friends
